app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<h2>Drivers!</h2>
+
+<div class="toggle">
+  <%= paginate @drivers, outer_window: 3 %>
+</div>
+
+<div class="driver_title">
```

Comment:
For CSS class names, we do not follow the Ruby naming conventions. For CSS this should be `driver-title` not `driver_title`.